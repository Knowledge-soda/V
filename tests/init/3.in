=(a, in())
print(v(a))
