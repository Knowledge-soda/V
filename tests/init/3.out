_start:
    NAME(id) BEGIN NAME(=) BEGIN NAME(a) SEP NAME(in) BEGIN END END NAME(print) BEGIN NAME(v) BEGIN NAME(a) END END END
